.prettierrc.yaml

```diff
@@ -0,0 +1,13 @@
+# .prettierrc
+printWidth: 160
```

Comment:
I am still opposed to this. Wider than github, makes reviewing changes in PRs more irritating.

Reply:
Let's see how it goes in practice I guess?